src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/ConversionResolution.vb

```diff
@@ -114,12 +115,19 @@ Namespace Microsoft.VisualBasic.CompilerServices
             VerifyTypeCodeEnum()
 
 #If DEBUG Then
-            VerifyForLoopWidestType()
+            VerifyForLoopWidestTypeInDebugMode()
 #End If
         End Sub
-
+        
 #If DEBUG Then
+        <UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification:="This method only gets called in DEBUG mode.")>
```

Comment:
Isn't it always? Since it is declared inside `#If DEBUG`?

Reply:
While you are correct, the whole reason for suppressing this here is so that I don't have to add a bunch of 
```
#If DEBUG Then
<RequiresUnreferencedCode("bla")>
#End If
```

Would that be preferred? Or is your comment more about the justification message itself?

Reply:
I was thinking - why not just add the suppression to the `VerifyForLoopWidestType` method itself. Doesn't seem necessary to introduce a new method `VerifyForLoopWidestTypeInDebugMode` just to add the suppression.

Reply:
Sure I can do that, the thinking was in case somebody else started calling this method should get the warning but you are correct that if they do it would be in DEBUG as well, so I like your suggestion, will go with that.